### PR TITLE
Update What is probability.rst

### PR DESCRIPTION
**Pull request by martscht**
Changes:
- First upload of chapter
- Change filename (Remove Bayes contra Hume)

Todo:
- Links to references are currently dead

### CODE REVIEW

**Comment by jona-sassenhagen** [comment]
Great - haven't had time to review in depth, but I'm looking forward to it!

**Comment by DejanDraschkow** [comment]
same here! but didnt we talk about maybe doing first submissions of theory/content chapters via gdocs? just for the first pass? and going via PRs for later edits and comments? 
still open to suggestions, but first drafts seem like a lot of PRs on PRs... 
what do you guys think?

**Comment by DejanDraschkow** [comment]
another thing we talked about was a "merge rule" - two people from the editor list _need_ to give an OK for merging! should we keep that rule?

**Comment by martscht** [comment]
> another thing we talked about was a "merge rule" - two people from the editor list _need_ to give an OK for merging! should we keep that rule?

I would keep this up.

> still open to suggestions, but first drafts seem like a lot of PRs on PRs...

My suggestion would be to have one editor read the draft of a chapter (if it's your own chapter, it would need to be one of the others) and then submit a PR. At least that's what I did with this chapter - so there is some direct communication with the author and then it simply goes into our internal loop.

**Comment by martscht** [comment]
*Bump* can this be committed? Or does anyone have feedback on what needs to be changed, before it goes live?

**Comment by jona-sassenhagen** [comment]
I will try to read it today.

**Comment by jona-sassenhagen** [comment]
`fold -w` to wrap at ~80c, hope this is ok.

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 1]
```diff
@@ -1,3 +1,801 @@
...
+and how could this be subjective in any way? This led to scientists treating 
+the two points of view in probability as diametrical (`McGrayne, 2011`_). 
+Soon mathematicians should take over the field of probability, bringing with 
+them the frequentist approach or objective probability and most terms, concepts 
+and techniques we use in statistics today (`Dienes, 2008`_).
+
+The most dominant name in this upcoming approach would become Sir Ronald Fisher 
+(1890-1962), a geneticist, who coined the terms ‘null hypothesis’ and 
+‘significance’, and suggested a p-value of .05 as an arbitrary level to judge a 
+result statistically significant (`Dienes, 2008`_). After working for years on
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst]: I would hedge this - Fisher is perhaps directly responsible for the overabundant *p < .05* standard, but it's not like he said, "a result is publishable if it meets the .05 p value, final word and law forever!". Rather he suggested it in *some* of his writings as a reasonable standard for considering that further investigating a phenomenon.

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 2]
```diff
@@ -1,3 +1,801 @@
...
+result statistically significant (`Dienes, 2008`_). After working for years on 
+randomization, sampling theory, experimental designs, tests of significance and 
+analysis of variance he published a revolutionary manual for 
+non-statisticians – Statistical Methods for Research Workers (`Fisher, 
+1925`_). This manual was a game changer and helped to turn the frequentist 
+approach into the state of the art for years to come.
+
+Aside from (or maybe because of) his smart mind, Fisher was in a constant 
+ad-hominem feud with his colleagues, especially Karl Pearson (1857-1936) – a 
+fanatical eugenicist and name-giver of the Pearson correlation, his son Egon
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst]: It seems unfair to call out Pearson for being an eugenicist and not also call out Fisher for being an eugenicist too, an [opponent of the UN declaration of equality of the 'races'](https://en.wikipedia.org/wiki/Ronald_Fisher#Race), a science denier, ...

Comment (jona-sassenhagen) [reply docs/chapter_phil/What is probability? Bayes contra Hume.rst]: I mean, either note it for both, or for neither.

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 3]
```diff
@@ -1,3 +1,801 @@
...
+randomization, sampling theory, experimental designs, tests of significance and 
+analysis of variance he published a revolutionary manual for 
+non-statisticians – Statistical Methods for Research Workers (`Fisher, 
+1925`_). This manual was a game changer and helped to turn the frequentist 
+approach into the state of the art for years to come.
+
+Aside from (or maybe because of) his smart mind, Fisher was in a constant 
+ad-hominem feud with his colleagues, especially Karl Pearson (1857-1936) – a 
+fanatical eugenicist and name-giver of the Pearson correlation, his son Egon 
+Pearson (1895-1980), a statistician, and his colleague Jerzy Neyman
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst:116]: It seems Neyman was a bit of an outlier in this regard :)

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 4]
```diff
@@ -1,3 +1,801 @@
...
+it will work ever again. Not unless you already assume induction, that is’ 
+(`Dienes, 2008`_, p.5). A historical fun fact that is closely related to the 
+swan argument, comes from the time that the British went to Australia. Guess 
+what they found? Of course, they found black swans.
+
+The second interesting mind here is Karl Popper, who wrote a lot about the 
+philosophy of science and what distinguishes science from non-science. Popper 
+argued against positivism and with it inductive thinking and his philosophy 
+was fallibilism. In a nutshell (because `Chapter 1.2`_ is dealing with this in 
+a much deeper way): You cannot say something is true, you can only falsify
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst]: For a narrow range of *something*s.

I would rephrase this. Popper is talking about a very specific kind of statement here.

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 5]
```diff
@@ -1,3 +1,801 @@
...
+swan argument, comes from the time that the British went to Australia. Guess 
+what they found? Of course, they found black swans.
+
+The second interesting mind here is Karl Popper, who wrote a lot about the 
+philosophy of science and what distinguishes science from non-science. Popper 
+argued against positivism and with it inductive thinking and his philosophy 
+was fallibilism. In a nutshell (because `Chapter 1.2`_ is dealing with this in 
+a much deeper way): You cannot say something is true, you can only falsify 
+statements. Popper agreed with Hume’s statements (e.g. `Popper, 1934`_). For 
+him a theory would in a best case scenario always be a guess, nothing more.
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst]: This also seems imprecise. I think it's pointing at the rather free-range nature of hypothesis generation as Popper sees it?

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 6]
```diff
@@ -1,3 +1,801 @@
...
+what they found? Of course, they found black swans.
+
+The second interesting mind here is Karl Popper, who wrote a lot about the 
+philosophy of science and what distinguishes science from non-science. Popper 
+argued against positivism and with it inductive thinking and his philosophy 
+was fallibilism. In a nutshell (because `Chapter 1.2`_ is dealing with this in 
+a much deeper way): You cannot say something is true, you can only falsify 
+statements. Popper agreed with Hume’s statements (e.g. `Popper, 1934`_). For 
+him a theory would in a best case scenario always be a guess, nothing more. 
+Maybe his reasoning had something to do with the fact that during his youth one
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst]: I don't like this "maybe". Popper says quite clearly in his biography 1. why he came to believe what he believed, 2. what the role of, e.g., the Eddington expedition was.

I would like this rephrased.

Comment (jona-sassenhagen) [reply docs/chapter_phil/What is probability? Bayes contra Hume.rst]: It is also a bit weird to discuss Popper *after* null hypothesis rejection testing. Popper was not a Bayesian; but NHST is often justified in Popperian terms.

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst [hunk 7]
```diff
@@ -1,3 +1,801 @@
...
+inference, and are sensitive to different things. You should be aware of your 
+research question and the kind of probability that helps you to find an answer 
+to this question. Do you need a black and white answer using objective 
+probability or do you need a continuous distribution of posterior beliefs using 
+subjective probability? Both probabilities come with a huge toolbox of 
+applicable statistical methods (and some of them are discussed by my colleagues 
+in this volume) and many of those methods can be used with both approaches. So 
+chose your tool and scientific approach to each question you ask very careful 
+and aware of the alternatives. I wish you a pleasant journey into the wonderful 
+world of statistics.
```
Comment (jona-sassenhagen) [inline docs/chapter_phil/What is probability? Bayes contra Hume.rst:637]: I don't like this last sentence :) 1. this is not a statistics book, 2. the tone is a bit too much. Maybe just cut it.

**Comment by jona-sassenhagen** [comment]
So there's my first pass. @martscht what do you think? I could make a PR to implement some of my suggestions?

**Comment by jona-sassenhagen** [comment]
Obviously, all in all I think it's a great chapter! I'm very happy that we have this.

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst (hunk 1 above)
Comment (martscht) [reply docs/chapter_phil/What is probability? Bayes contra Hume.rst]: Maybe rephrase to: 

>  and 'significance' (Dienes, 2008). Though often deemed the originator of the tradition of using p < .05 as an arbitrary level to judge results as statistically significant, Fisher (1926, p. 85) was open in suggesting that each researcher should select a threshold in relation to the specific study. As a matter of fact, Gigerenzer, Krauss, and Vitouch (2004, p. 392) suggest that "Fisher thought that using a routine 5% level of significance indicated lack of statistical thinking".

File: docs/chapter_phil/What is probability? Bayes contra Hume.rst (hunk 2 above)
Comment (martscht) [reply docs/chapter_phil/What is probability? Bayes contra Hume.rst]: agreed, would remove it for Pearson.

**Comment by martscht** [comment]
> So there's my first pass. @martscht what do you think? I could make a PR to implement some of my suggestions? 

This version is on my fork, you can just edit and commit directly to it there and then we will update this PR. I will also send the author a link to this discussion and the suggestions, in case he does not agree with some of them.

**Comment by martscht** [comment]
The author gave us the go-ahead to include the changes. Thanks for your detailed feedback @jona-sassenhagen !

**Comment by martscht** [comment]
Did not mean to delete this on my repo.

**Comment by jona-sassenhagen** [comment]
:D

Ok, so I will commit directly.

**Comment by martscht** [comment]
This one is done, right? Can we commit this, so I can tidy up a bit?

**Comment by jona-sassenhagen** [comment]
Sorry, I know you're eager to move forward here, but I would prefer to reorganise the parts about Popper first still!

**Comment by jona-sassenhagen** [comment]
Sorry for being way late. I am getting back to this.

**Comment by jona-sassenhagen** [comment]
I added to the Popper section. With that, it's good to go from my end.

**Comment by jona-sassenhagen** [comment]
@aylinsgl can you merge?
